app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -78,9 +78,8 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
         continue;
       }
 
-      final String defaultBranch = Config.defaultBranch(slug);
-      if (pr.base!.ref != defaultBranch) {
-        log.fine('This change is not staged to land on $defaultBranch, skipping.');
+      if (!Config.doesSkiaGoldRunOnBranch(slug, pr.base!.ref)) {
```

Comment:
> When someone is working on a release branch, and the golden test fails, blocking a change from landing, how do they resolve it?

If a cherry-pick on a release branch breaks a golden test, fixing that would depend on why it was broken.  It could mean:
1) An intermediate cherry-pick is missing and needs to also be merged in
1) It's an integration problem where a fix would have to be devised:
   1) decide not to cherry-pick
   1) author a PR for the release branch
   1) author a PR on the main branch and cherry-pick on the release branch

> And has this been communicated to the team so folks know what to do?

Yep, there was design doc that was worked on with the infra team: [go/flutter-engine-goldens-workflow](http://goto.google.com/flutter-engine-goldens-workflow), there is also the documentation I linked in the PR description.  The linked issue will be the basis for more concrete instructions once we've connected all the dots.

Reply:
SGTM! Thanks for the references. If anyone comes my way I'll direct them there!